src/main/scala/sha3.scala

```diff
@@ -21,14 +22,19 @@ abstract class SimpleRoCC()(implicit p: Parameters) extends RoCC()(p)
     // Set this true to trigger an interrupt on the processor (please refer to supervisor documentation)
 
   //a simple accelerator doesn't use imem or page tables
-  io.imem.acquire.valid := Bool(false)
-  io.imem.grant.ready := Bool(true)
-  io.dmem.head.acquire.valid := Bool(false)
-  io.dmem.head.grant.ready := Bool(false)
-  io.iptw.req.valid := Bool(false)
-  io.dptw.req.valid := Bool(false)
-  io.pptw.req.valid := Bool(false)
-  io.mem.invalidate_lr := Bool(false)
+
+  //Old Format
+  //io.imem.acquire.valid := Bool(false)
+  //io.imem.grant.ready := Bool(false)
+  //io.imem.finish.valid := Bool(false)
+  //io.iptw.req.valid := Bool(false)
+  //io.dptw.req.valid := Bool(false)
+  //io.pptw.req.valid := Bool(false)
+
+  //New Format
+  io.autl.acquire.valid := Bool(false)
+  io.autl.grant.ready := Bool(false)
```

Comment:
This seems like a change from before where imem.grant.ready was tied to true not false